ariadne/cardano/src/Ariadne/System/FileMode.hs

```diff
@@ -0,0 +1,47 @@
+module Ariadne.System.FileMode
+       ( ensureModeIs600
+       ) where
+
+import System.Info (os)
+import System.Wlog (logWarning, usingLoggerName, CanLog)
+
+import qualified System.Posix.Files as PSX
+import qualified System.Posix.Types as PSX (FileMode)
+
+-- | It's common practice to make keyfiles readable/writable for user only.
+-- For now keyfile is read-only
```

Comment:
> For now keyfile is read-only

What does it mean?

Reply:
Corrected it to `-- For now keyfile is read and write only`